src/transform/Transformer.ts

```diff
@@ -251,25 +251,45 @@ class Transformer {
   convertImportDeclaration(node: ts.ImportDeclaration | ts.ImportEqualsDeclaration) {
     if (ts.isImportEqualsDeclaration(node)) {
       // assume its like `import default`
-      if (!ts.isExternalModuleReference(node.moduleReference)) {
-        throw new UnsupportedSyntaxError(node, "ImportEquals should have a literal source.");
+      if (ts.isExternalModuleReference(node.moduleReference)) {
+        this.pushStatement(
+          withStartEnd(
+            {
+              type: "ImportDeclaration",
+              specifiers: [
+                {
+                  type: "ImportDefaultSpecifier",
+                  local: createIdentifier(node.name),
+                },
+              ],
+              source: convertExpression(node.moduleReference.expression) as any,
+            },
+            node,
+          ),
+        );
+        return;
+      } else if (ts.isQualifiedName(node.moduleReference)) {
+        // TODO: I *think* this just ensures that the statement doesnt get tree-shaken out of existence, even though there's no real variable declaration here that makes it into the final build
+        this.createDeclaration(node, node.name);
+        this.pushStatement(
```

Comment:
This indeed feels a bit out of place here. Maybe merging this with the above `if` will also work, not sure though.